src/biokbase/narrative/jobs/jobmanager.py

```diff
@@ -539,20 +524,23 @@ def retry_jobs(self, job_id_list: List[str]) -> dict:
                 results_by_job_id[job_id]["retry"] = job_states[retry_id]
             if "error" in result:
                 results_by_job_id[job_id]["error"] = result["error"]
-        for job_id in error_ids:
-            results_by_job_id[job_id] = {
-                "job_id": job_id,
-                "job": get_error_output_state(job_id),
-                "error": DOES_NOT_EXIST,
+        return self.add_errors_to_results(results_by_job_id, error_ids)
+
+    def add_errors_to_results(self, results: dict, error_ids: List[str]) -> dict:
+        """
+        Add the generic "not found" error for each job_id in error_ids
+        """
+        for error_id in error_ids:
+            results[error_id] = {
+                "job_id": error_id,
+                "error": f"Cannot find job with ID {error_id}",
```

Comment:
There was a mishmash of ways to handle not found IDs, so to save from having to implement different code for each in the FE, I've standardised them all to this structure.